docs/admin/oauth2/create-okta-client.md

```diff
@@ -0,0 +1,55 @@
+# Create An Application in Okta
+
+This example uses an Okta development account to create an application that will represent a Hopsworks client in the 
+identity provider. To create a developer account go to [Okta developer](https://developer.okta.com/signup/).
+
+After creating a developer account register a client by going to _Applications_ and click on **Create App Integration**.
+
+  <figure>
+    <a  href="../../../assets/images/admin/oauth2/okta.png">
+      <img src="../../../assets/images/admin/oauth2/okta.png" alt="Okta Applications" />
+    </a>
+    <figcaption>Okta Applications</figcaption>
+  </figure>
+
+This will open a popup as shown in the figure below. Select **OIDC** as _Sign-in-method_ and **Web Application** as 
+_Application type_ and click next.
+  <figure>
+    <a  href="../../../assets/images/admin/oauth2/create-new-app.png">
+      <img src="../../../assets/images/admin/oauth2/create-new-app.png" alt="Create New Application" />
+    </a>
+    <figcaption>Create new Application</figcaption>
+  </figure>
+
+Give your application a name and select **Client credential** as _Grant Type_. If you want to limit who can access 
+your Hopsworks cluster select _Limit access to selected groups_ and select group(s) you want to give access to. Here 
+we will allow everyone in the organization to access the cluster.
```

Comment:
I gess the redirect urls need to be set too. There should be some text about it here.